editor/example/index.ts

```diff
@@ -20,149 +20,154 @@ import { EX_SPEC_BAND } from './vertical-band';
+    BASIC_SEMANTIC_ZOOM: {
         name: 'Basic Example: Basic Idea of Semantic Zoom',
         id: 'BASIC_SEMANTIC_ZOOM',
         spec: EX_SPEC_BASIC_SEMANTIC_ZOOM,
         hidden: true
     },
-    {
+    MARK_DISPLACEMENT: {
         name: 'Basic Example: Mark Displacement',
         id: 'MARK_DISPLACEMENT',
         spec: EX_SPEC_MARK_DISPLACEMENT
     },
-    {
+    CIRCULAR_OVERVIEW_LINEAR_DETAIL: {
         name: 'Basic Example: Circular Overview + Linear Detail Views',
         id: 'CIRCULAR_OVERVIEW_LINEAR_DETAIL',
         spec: EX_SPEC_CIRCULAR_OVERVIEW_LINEAR_DETAIL
     },
-    {
+    SEQUENCE: {
         name: 'Multi-Scale Sequence Track',
         id: 'SEQUENCE',
         spec: EX_SPEC_SEQUENCE_TRACK
     },
-    {
+    SEMANTIC_ZOOM: {
         name: 'Multi-Scale Clinvar Lollipop Plot',
         id: 'SEMANTIC_ZOOM',
         spec: EX_SPEC_CLINVAR_LOLLIPOP
     },
-    {
+    CYTOBANDS: {
         name: 'Ideograms',
         id: 'CYTOBANDS',
         spec: EX_SPEC_CYTOBANDS
     },
-    {
+    GENE_ANNOTATION: {
         name: 'Custom Gene Annotation',
         id: 'GENE_ANNOTATION',
         spec: EX_SPEC_GENE_ANNOTATION
     },
-    {
+    MATRIX_HFFC6: {
         name: 'Comparative Matrices (Micro-C vs. Hi-C)',
         id: 'MATRIX_HFFC6',
         spec: EX_SPEC_MATRIX_HFFC6
     },
-    {
+    CIRCOS: {
         name: 'Circos',
         id: 'CIRCOS',
         spec: EX_SPEC_CIRCOS
     },
-    {
+    'Circular Range': {
         name: 'Circular Range (Inspired By Weather Radials)',
         id: 'Circular Range',
         spec: EX_SPEC_CIRCULR_RANGE,
         hidden: true
     },
-    {
+    SARS_COV_2: {
         name: 'SARS-CoV-2',
         id: 'SARS_COV_2',
         spec: EX_SPEC_SARS_COV_2
     },
-    {
+    CORCES_ET_AL: {
         name: 'Corces et al. 2020',
         id: 'CORCES_ET_AL',
         spec: EX_SPEC_CORCES_ET_AL
     },
-    {
+    CIRCULAR_BETWEEN_BANDS: {
         name: 'Circular Between Bands',
         id: 'CIRCULAR_BETWEEN_BANDS',
         spec: EX_SPEC_CIRCOS_BETWEEN_LINK,
         underDevelopment: true
     },
-    {
+    GREMLIN: {
         name: "Gremlin (O'Brien et al. 2010)",
         id: 'GREMLIN',
         spec: EX_SPEC_GREMLIN,
         underDevelopment: true
     },
-    {
+    GIVE: {
         name: 'GIVE (Cao et al. 2018)',
         id: 'GIVE',
         spec: EX_SPEC_GIVE,
         underDevelopment: true
     },
-    {
+    CANCER_VARIANT: {
         name: 'Breast Cancer Variant (Staaf et al. 2019)',
         id: 'CANCER_VARIANT',
         spec: EX_SPEC_CANCER_VARIANT_PROTOTYPE,
         underDevelopment: true
     },
-    {
+    BAM_PILEUP: {
         name: 'BAM file pileup tracks',
         id: 'BAM_PILEUP',
         spec: EX_SPEC_PILEUP,
         underDevelopment: true
     },
-    {
+    TEMPLATE: {
         name: 'Track Template',
         id: 'TEMPLATE',
         spec: EX_SPEC_TEMPLATE,
         underDevelopment: true
-    }
-].filter(d => !d.hidden);
+    },
+    ...DOC_EXAMPLES
```

Comment:
What do you think of adding doc-related examples (i.e., examples in `DOC_EXAMPLES`) directly here so that we can maintain examples in a single file? Perhaps we can add a property, like `doc: boolean` if needed.

Reply:
Sounds great~ Maybe no need to add another property, I can just assign their keys as `doc_xxx` and set `hidden: true`